basics/transforming-dto-data.md

```diff
@@ -56,6 +56,9 @@ $dto->toModel(\App\Models\User::class);
 // }
 ```
 
+### Custom transforming
+If you want to build your own DataTransformer, you can use `buildDataForExport()` to retrieve the validated data with your custom mappings.
```

Comment:
Update this to:

If you want to build your own Data Transformer, you can use the `buildDataForExport()` method to retrieve the validated data with your custom mappings.

And add this example below it:

```php
public function toObject(): object
{
    return (object) $this->buildDataForExport();
}
```

Reply:
Thanks for your help. I changed it to your suggestion.